projects.go

```diff
@@ -58,17 +59,50 @@ type Project struct {
 	SharedRunners        bool       `json:"shared_runners_enabled"`
 }
 
+func append(slice []*Project, elements ...*Project) []*Project {
+	n := len(slice)
+	total := len(slice) + len(elements)
+	if total > cap(slice) {
+		// Reallocate. Grow to 1.5 times the new size, so we can still grow.
+		newSize := total*3/2 + 1
+		newSlice := make([]*Project, total, newSize)
+		copy(newSlice, slice)
+		slice = newSlice
+	}
+	slice = slice[:total]
+	copy(slice[n:], elements)
+	return slice
+}
+
 func projects(u string, g *Gitlab) ([]*Project, error) {
-	url := g.ResourceUrl(u, nil)
 
 	var projects []*Project
-
-	contents, err := g.buildAndExecRequest("GET", url, nil)
-	if err == nil {
-		err = json.Unmarshal(contents, &projects)
+	var page int64 = 1
+	for {
```

Comment:
@plouc yep, you are right, i think i can rewrite a pr for missing  pagination capabilities.